courses/cunix/ex01/src/my_strlen.c

```diff
@@ -0,0 +1,9 @@
+unsigned int my_strlen(char *str)
+{
+  char *array = str;
+  while (*str != 0)
+  {
+    str = str + 1; 
+  }
+  return str - array;
+}
```

Comment:
for the future please put a newline at the end of the file

1 point for this task